readme.md

```diff
@@ -62,6 +63,10 @@ Contributions welcome. Add links through pull requests or create an issue to sta
 - [Juggling TV](http://juggling.tv/) - A juggling specific video site.
 - [Manipeo](http://manipeo.com/) - A user submitted juggling video site.
 
+## Games
+
+- [siteswap anagrams](http://siteswapgame.herokuapp.com/) - puzzle game for calculating siteswap anagrams
```

Comment:
For consistency, can you change this to 
[Siteswap Anagrams](http://siteswapgame.herokuapp.com/) - A puzzle game for calculating siteswap anagrams.

Reply:
👍 done